src/sentry/seer/signed_seer_api.py

```diff
@@ -21,13 +22,20 @@ def make_signed_seer_api_request(
     if timeout:
         timeout_options["timeout"] = timeout
 
-    return connection_pool.urlopen(
-        "POST",
-        path,
-        body=body,
-        headers={"content-type": "application/json;charset=utf-8", **auth_headers},
-        **timeout_options,
-    )
+    with metrics.timer(
+        "seer.request_to_seer",
+        tags={
+            # Pull off query params, if any
+            "endpoint": path.split("?")[0],
```

Comment:
should be able to use `urlparse(url1).path` which might be a tad safer

Reply:
Didn't know you could use `urlparse` when all you had was a path! Just tested it out and it seems to work, though, so I made the change.